h2/src/test/org/h2/test/scripts/functions/string/regex-replace.sql

```diff
@@ -15,6 +15,18 @@ select regexp_replace('Sylvain', 'S..', 'TOTO', 'mni');
 set mode oracle;
 > ok
 
+select regexp_replace('.1.2.3.4', '[^0-9]', '', 1, 0);
```

Comment:
You need to additionally test your implementation with 5 and 6 arguments and with empty strings in other arguments too.

If your IDE has some built-in code coverage tool, run it with `TestScript` class and make sure that all new conditions are covered with tests.